x-pack/plugins/stack_alerts/server/alert_types/geo_containment/tests/geo_containment.test.ts

```diff
@@ -53,7 +52,7 @@ describe('geo_containment', () => {
     it('should correctly transform expected results', async () => {
       const transformedResults = transformResults(
         // @ts-ignore
-        (sampleAggsJsonResponse.body as unknown) as SearchResponse<unknown>,
+        sampleAggsJsonResponse.body,
```

Comment:
the line has `// @ts-ignore` directive, so the manual casting is unnecessary. and it's wrong anyway.